pandas/core/tools/timedeltas.py

```diff
@@ -113,9 +113,10 @@ def _coerce_scalar_to_timedelta_type(r, unit="ns", errors="raise"):
 
     try:
         result = Timedelta(r, unit)
-    except ValueError:
+    except ValueError as e:
```

Comment:
can you call this "err" instead of "e".  i really dislike 1-letter variable names

Reply:
even here. ok!)